src/gwt/src/org/rstudio/studio/client/workbench/views/environment/dataimport/DataImport.java

```diff
@@ -308,6 +308,40 @@ public void execute(final String formatString)
             }
          });
    }
+
+   private void promptForFactorString(
+      String title,
+      final Operation complete,
+      final String columnName,
+      final String columnType)
+   {
+      globalDisplay_.promptForText(
+         title,
+         "Please a comma separated list of factors",
```

Comment:
Agree that `selectize.js` would be a plus here but indeed to late in the release. +1 on "insert"